PWGDQ/Tasks/tableReader.cxx

```diff
@@ -348,6 +369,197 @@ struct AnalysisMuonSelection {
   PROCESS_SWITCH(AnalysisMuonSelection, processDummy, "Dummy function", false);
 };
 
+struct AnalysisPrefilterSelection {
+  Produces<o2::aod::DalitzBitsReduced> dalitzbits;
+
+  // Configurables
+  Configurable<std::string> fConfigEventCuts{"cfgEventCuts", "eventStandardNoINT7", "Event selection"};
+  Configurable<std::string> fConfigDalitzTrackCuts{"cfgDalitzTrackCuts", "", "Dalitz track selection cuts, separated by a comma"};
+  Configurable<std::string> fConfigDalitzPairCuts{"cfgDalitzPairCuts", "", "Dalitz pair selection cuts"};
+  Configurable<bool> fQA{"cfgQA", false, "QA histograms"};
+
+  Filter filterEventSelected = aod::dqanalysisflags::isEventSelected == 1;
+  Filter filterBarrelTrackSelected = aod::dqanalysisflags::isBarrelSelected > 0; // Jpsi candidates AND dalitz candidates
+
```

Comment:
In the AnalysisPrefilterSelection we may also want to do some QA for dalitz (the same as we did in dalitzSelection.cxx), and then in this case we would want to fill in histograms with all dalitz pairs (and not only tracks we veto, even if at the end, in the table, we only fill bits corresponding to the regular tracks we want to veto). In the case we do not want qa, there is a continue argument if none of the tracks is a Jpsi daughter. 
It seemed to me that the partitions made it a bit more complex if we wanted to keep both functionalities ("veto +QA" or "veto only").
Another thing I noticed with the partitions is that the grouping per event with sliceByCached take some constant time for every event, and the grouping time is the dominant one since for most events, there is less than two candidates and then no pairing.